cmd/commands/server/factory.go

```diff
@@ -137,6 +139,31 @@ func NewCommandWith(
 		checkOpenvpn: func() error {
 			return openvpn.CheckOpenvpnBinary(options.OpenvpnBinary)
 		},
+		openvpnServiceAddress: func() (string, error) {
+			publicIP, err := ipResolver.GetPublicIP()
+			if err != nil {
+				return "", err
+			}
+
+			outboundIP, err := ipResolver.GetOutboundIP()
+			if err != nil {
+				return "", err
+			}
+
+			if publicIP != outboundIP {
+				forwardInfo := fmt.Sprintf("%s:%v -> %s:%v", publicIP, options.OpenvpnPort, outboundIP, options.OpenvpnPort)
+				log.Warnf(
+					`WARNING: It seems that publicaly visible ip: [%s] does not match your local machines ip: [%s]. 
+You should probaly need to do port forwarding on your router: %s.`,
```

Comment:
`..on your router: PUBLIC_IP:1194 -> MACHINE_IP:1194 e.g. %s:1194->%s:1194`
and you can put example values from detected values

Reply:
Hmm ok now it looks like this:
2018-07-12T11:45:11.643023557 [Warn] WARNING: It seems that publicaly visible ip: [46.255.210.150] does not match your local machines ip: [192.168.0.131]. 
You should probaly need to do port forwarding on your router: 46.255.210.150:1194 -> 192.168.0.131:1194.

Are you suggesting to expand message even more like "... on your router: PUBLIC_IP:1194 -> MACHINE_IP:1194 e.g. 46.255.210.150:1194 -> 192.168.0.131:1194." ?

Reply:
Sorry, did not see that it's already like that

Reply:
👍